docs/Guides-Renderer.md

```diff
@@ -0,0 +1,168 @@
+---
+id: guides-renderer
+layout: docs
+category: Guides
+permalink: docs/guides-renderer.html
+next: guides-ready-state
+---
+
+So far, we've covered two pieces that each contribute to declaring data:
+
+ - **Relay.Route** lets us declare query roots.
+ - **Relay.Container** lets components declare fragments.
+
+To use these pieces to construct a full-fledged GraphQL query that we can send to the server to fetch data, we need to use the **Relay.Renderer**.
+
+## Component and Route
+
+**Relay.Renderer** is a React component that, given a `Container` and a `route` or `queryConfig`, attempts to fulfill the data required in order to render an instance of `Container`.
```

Comment:
Maybe don't mention routes. We're trying to get rid of them in favor of query configs.